modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApiController.mustache

```diff
@@ -27,12 +27,12 @@ import swagger.SwaggerUtils.ApiAction;
 {{#operations}}
 public class {{classname}}Controller extends Controller {
 
-    private final {{classname}}ControllerImp imp;
+    {{^controllerOnly}}private final {{classname}}ControllerImp imp;{{/controllerOnly}}
```

Comment:
@JFCote next time you may want to use the following instead to avoid the leading spaces:
```
    {{^controllerOnly}}
    private final {{classname}}ControllerImp imp;
    {{/controllerOnly}}
```